### DISCUSSION TITLE
How to disable "Unwatch suggestions" beta?

### DISCUSSION THREAD

**Discussion by oliversalzburg** [community/community#17505] [Other Feature Feedback, Questions, & Ideas]
Whenever I want to read my notifications, GitHub feels the need to tell me that I'm obviously overwhelmed by my notifications and that I'm probably not managing my notifications settings properly.

I beg to differ on GitHub's assessment regarding my ability to manage notifications and watch settings for all my repositories.

I couldn't find any way to disable this behavior and I'm constantly annoyed by these nag banners over and over again. I don't understand what this call to action is trying to achieve even. Make me ignore notifications? And how is that an improvement?

![image](https://user-images.githubusercontent.com/1658949/170995345-0fedbbc6-0943-4cf5-a6ae-19c3ffe0e493.png)

Please stop this constant hand-holding or at least give users an option to control this behavior.

**Comment by alexjpwalker** [comment]
2 years in I am still yet to see any benefit from this feature.

How is _another_ notification, that flags up repositories seemingly at random, that can't be disabled, a solution to "too many notifications"?

In its current form, it is unfortunately a subpar fix to a problem that never existed.
